cabra/src/pages/components/TaskListItem.tsx

```diff
@@ -76,6 +85,13 @@ export default function TaskListItem({ task }: Props) {
             </Link>
           </Card>
         </div>
+        {task.relationTypes.length > 0 && (
+          <Card tw="flex gap-2">
+            {task?.relationTypes.map((type) => (
+              <TaskTypeIcon key={type} type={type} />
+            ))}
+          </Card>
+        )}
```

Comment:
Got some remarks about this div.
It is visible conditionally, and I think it's bad. I think we should render it always, maybe with some faded empty containers to show the task is unrelated.
Also not-constant task size is vary bad.
What about this concept (it includes resizing all cards in the task a bit):
![image](https://user-images.githubusercontent.com/72392522/168015175-1f8d6f70-56ea-4e68-b3a7-d7cc3b664b96.png)